admin-manual/installation-setup/installation/scripts/am19-bionic-deb.sh

```diff
@@ -31,6 +31,17 @@ sudo apt-get install -y archivematica-storage-service
 sudo rm -f /etc/nginx/sites-enabled/default
 sudo ln -s /etc/nginx/sites-available/storage /etc/nginx/sites-enabled/storage
 
+#Fix for mysql UTF8 problems
```

Comment:
You need to create the /etc/mysql/conf.d dir first:
<pre>
sudo mkdir -p /etc/mysql/conf.d
</pre>